soc/arm/st_stm32/common/Kconfig.defconfig.series

```diff
@@ -7,6 +7,11 @@
 
 if SOC_FAMILY_STM32
 
+config STM32_LPTIM_TIMER
```

Comment:
Since this is not yet supported in all series (to prevent wrong usages), please move this in:
* soc/arm/st_stm32/stm32l4/Kconfig.defconfig.series
and
* soc/arm/st_stm32/stm32wb/Kconfig.defconfig.series

Also we should remove equivalent config in boards configuration (disco_l475_iot1 and nucleo_wb5rg).